### PR TITLE
Add icons

### PR DESCRIPTION
**Pull request by toddmoy**
* Health
* Support
* Gift (filled, outline)

---

image

### CODE REVIEW

File: src/gift-fill.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path d="M11 14v8H5c-1.057 0-1.922-.482-1.995-1.813L3 20v-6h8zm10 0v6c0 1.409-.82 1.952-1.857 1.997L19 22h-6v-8h8zM5 2h1c2.89 0 4.87 1.239 6 3.28C13.13 3.238 15.11 2 18 2h1v1c0 .891-.211 2.158-.856 3.447a6.645 6.645 0 01-1.064 1.555L20 8a2 2 0 011.995 1.85L22 10v2h-9V9h-2v3H2v-2a2 2 0 011.85-1.995L4 8l2.92.002a6.645 6.645 0 01-1.064-1.555c-.599-1.197-.824-2.374-.853-3.25L5 3V2h1zm11.86 2.09c-1.568.262-2.584 1.093-3.216 2.357-.185.37-.327.763-.432 1.163l-.072.3c1.568-.262 2.584-1.093 3.216-2.357.23-.463.396-.96.504-1.464zm-9.72 0c.108.503.273 1 .504 1.463.597 1.193 1.537 2.002 2.96 2.309l.256.049-.072-.301a6.005 6.005 0 00-.432-1.163C9.724 5.183 8.708 4.352 7.14 4.09z" id="a"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a"/></svg>
```
Comment (roarytubbs) [inline src/gift-fill.svg]: - Please remove unnecessary `<defs>` `</defs>`
- Remove the`id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a"/>`

File: src/gift-outline.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path d="M5 22a2 2 0 01-1.995-1.85L3 20v-6H2v-4a2 2 0 011.85-1.995L4 8l2.92.002a6.645 6.645 0 01-1.064-1.555c-.599-1.197-.824-2.374-.853-3.25L5 3V2h1c2.89 0 4.87 1.239 6 3.28C13.13 3.238 15.11 2 18 2h1v1c0 .891-.211 2.158-.856 3.447a6.645 6.645 0 01-1.064 1.555L20 8a2 2 0 011.995 1.85L22 10v4h-1v6a2 2 0 01-1.85 1.995L19 22H5zm6-8H5v6h6v-6zm8 0h-6v6h6v-6zm-8-4H4v2h7v-2zm9 0h-7v2h7v-2zm-3.14-5.91c-1.568.262-2.584 1.093-3.216 2.357-.185.37-.327.763-.432 1.163l-.072.3c1.568-.262 2.584-1.093 3.216-2.357.23-.463.396-.96.504-1.464zm-9.72 0c.108.503.273 1 .504 1.463.597 1.193 1.537 2.002 2.96 2.309l.256.049-.072-.301a6.005 6.005 0 00-.432-1.163C9.724 5.183 8.708 4.352 7.14 4.09z" id="a"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a"/></svg>
```
Comment (roarytubbs) [inline src/gift-outline.svg]: - Please remove unnecessary `<defs>` `</defs>`
- Remove the `id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a"/>`

File: src/health.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path id="a" d="M7.5 3.5l5 10.5L16 8h4V6h-5l-2.5 4.5L7.5 0 4 6H0v2h5z"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 5)"/></svg>
```
Comment (roarytubbs) [inline src/health.svg]: - Please remove unnecessary `<defs>` `</defs>`
- Remove the `id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 5)"/>`

File: src/support.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path d="M10 0C4.477 0 0 4.477 0 10s4.477 10 10 10 10-4.477 10-10C19.994 4.48 15.52.006 10 0zm7.857 7.222a.208.208 0 01-.196.278h-3.214a.21.21 0 01-.178-.1 5.042 5.042 0 00-1.667-1.667.21.21 0 01-.102-.18V2.34a.21.21 0 01.278-.197 8.376 8.376 0 015.079 5.08v-.001zM10 13.333a3.333 3.333 0 110-6.666 3.333 3.333 0 010 6.666zM7.222 2.143a.21.21 0 01.278.197v3.213a.21.21 0 01-.1.179c-.68.416-1.25.987-1.667 1.666a.21.21 0 01-.18.102H2.339a.21.21 0 01-.172-.083.206.206 0 01-.026-.19 8.376 8.376 0 015.08-5.084zM2.143 12.778a.207.207 0 01.196-.278h3.214a.21.21 0 01.178.1c.416.68.987 1.25 1.667 1.667a.21.21 0 01.1.178v3.213a.208.208 0 01-.279.196 8.376 8.376 0 01-5.076-5.076zm10.635 5.08a.208.208 0 01-.278-.196v-3.214a.21.21 0 01.1-.178 5.042 5.042 0 001.667-1.667.21.21 0 01.178-.1h3.213a.208.208 0 01.193.278 8.376 8.376 0 01-5.073 5.077z" id="a"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 2)"/></svg>
```
Comment (roarytubbs) [inline src/support.svg]: - Please remove unnecessary `<defs>` `</defs>`
- Remove the `id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 2)"/>`

**Comment by toddmoy** [comment]
good catch - removed those crufty bits!

**Comment by thrashr888** [comment]
Is this still active?

**Comment by toddmoy** [comment]
@thrashr888 yep. Totally forgot about this. Merging it right meow.

**Pull request merged**